websockets/speedups.c

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2009 Facebook
```

Comment:
I really don't want to take responsibility for shipping code copyrighted by Facebook either.

They have lawyers and money. I don't.